Marlin/src/module/planner.cpp

```diff
@@ -213,7 +213,7 @@ xyze_float_t Planner::previous_speed;
 float Planner::previous_nominal_speed_sqr;
 
 #if ENABLED(DISABLE_INACTIVE_EXTRUDER)
-  last_move_t Planner::g_uc_extruder_last_move[EXTRUDERS] = { 0 };
+  last_move_t Planner::g_uc_extruder_last_move[E_STEPPERS] = { 0 };
```

Comment:
This feature might need a refresh. The `SWITCHING_EXTRUDER` is now able to use up to 3 motors, with each motor pertaining to 2 extruder indexes. But I don't think the use of the `g_uc_extruder_last_move` array is correct for `SWITCHING_EXTRUDER`, particularly with the `ENABLE_ONE_E` macro below. For the case of a switching extruder, the incoming `e` value should be divided by 2 to get the actual stepper index.

Reply:
I'll check it to see if I can manate it